manifests/rbac.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dao
+  namespace: dao
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dao
+rules:
+- apiGroups:
+  - "dao.mayadata.io"
+  resources:
+  - "*"
+  verbs:
+  - "*"
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: dao
+subjects:
+- kind: ServiceAccount
+  name: dao
+  namespace: dao
+roleRef:
+  kind: ClusterRole
+  name: dao
+  apiGroup: rbac.authorization.k8s.io
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: aggregate-dao-view
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
```

Comment:
Are we using these annotations?

Reply:
So basically -- the admin, view and edit roles are now having the roles defined in the current `aggregate-dao-view` . 
So if we only aggregate the roles to `rbac.authorization.k8s.io/aggregate-to-view: "true"` -- will it not be goog enough ?